Widgets/Widgets/LockscreenSearchWidget.swift

```diff
@@ -0,0 +1,127 @@
+import SwiftUI
+import WidgetKit
+import WMF
+import WMFComponents
+
+// MARK: - Widget
+
+struct LockscreenSearchWidget: Widget {
+    private let kind: String = WidgetController.SupportedWidget.lockscreenSearch.identifier
+    
+    public var body: some WidgetConfiguration {
+        StaticConfiguration(kind: kind, provider: LockscreenSearchProvider(), content: { entry in
+            LockscreenSearchWidgetView(entry: entry)
+        })
+        .configurationDisplayName(LockscreenSearchWidget.LocalizedStrings.widgetTitle)
+        .description(LockscreenSearchWidget.LocalizedStrings.widgetDescription)
+        .supportedFamilies([.accessoryCircular, .accessoryRectangular, .accessoryInline])
+        .contentMarginsDisabled()
+        .containerBackgroundRemovable(false)
```

Comment:
I still get the same background error, you need to use `containerBackground` here and use `contentMargins(.disabled)`